web/srv/api_handlers.go

```diff
@@ -39,6 +41,14 @@ var (
 		ReadBufferSize:  maxMessageSize,
 		WriteBufferSize: maxMessageSize,
 	}
+
+	// Checks whose description matches the following regexp won't be included
+	// in the handleApiCheck output. In the context of the dashboard, some
+	// checks like cli or kubectl versions ones may not be relevant.
+	//
+	// TODO(tegioz): use more reliable way to identify the checks that should
+	// not be displayed in the dashboard (hint anchor is not unique).
+	excludedChecksRE = regexp.MustCompile(`(?i)cli|(?i)kubectl`)
```

Comment:
Hi @scottcarol,

To avoid introducing changes to the `healthcheck` package I'm omitting the checks not relevant to the dashboard on the dashboard web server using a regular expression over the checker description. I'm not very happy with this though.

I think a better solution would be to introduce a new option in `healthcheck.Options` to indicate the health checker instance will be used for the dashboard, as well as a new flag to `healthcheck.checker` to mark individual checks to be skipped if the health checker instance is meant to be used for the dashboard. That way unneeded tests wouldn't even be run and no unreliable filtering would be needed on the dashboard web server. Happy to make this change if you guys are happy with that approach (or happy to implement any other solution you prefer).

Reply:
Moved to #3670